internal/driver/mobile/canvas.go

```diff
@@ -407,7 +407,7 @@ func (c *canvas) waitForDoubleTap(co fyne.CanvasObject, ev *fyne.PointEvent, tap
 		c.touchCancelFunc = nil
 		c.touchLastTapped = nil
 		c.touchCancelLock.Unlock()
-	})
+	}, true)
```

Comment:
should this function call just be converted to `fyne.DoAsync`?

Reply:
I'm not sure - that is going out of the driver to a utility function which does thread checks then back into the driver code - feels inefficient. Either way wouldn't it be the Sync/wait/fyne.Do version?